services/app-api/src/postgres/contractAndRates/findContractWithHistory.ts

```diff
@@ -11,9 +11,9 @@ import { convertPrismaErrorToStoreError, StoreError } from '../storeError'
 async function findContractWithHistory(
     client: PrismaTransactionType,
     contractID: string
-): Promise<ContractType | StoreError | Error> {
```

Comment:
Open question wether this should be `| NotFoundError` I'm fine with it not for now but it should probably be documented somewhere, maybe in the comment above?

Reply:
I think it should, I just forgot to add it to the return type. My reason for adding it is because we are specifially handling not found errors and returning `NotFoundError`.